### PR TITLE
(Nearly) complete MainMenuScreen and Container mappings

### PR DESCRIPTION
**Pull request by holly-hacker**
Completes most of MainMenuScreen and ContainerScreen, adds CubeMapRenderer and RotatingCubeMapRenderer (panorama background) and adds some minor missing mappings for arguments.

I don't usually program in Java, so please do tell me if I screwed up some naming schemes.

### CODE REVIEW

File: mappings/net/minecraft/client/gui/ContainerScreen.mapping [hunk 1]
```diff
@@ -1,4 +1,17 @@
 CLASS cwe net/minecraft/client/gui/ContainerScreen
+	FIELD A touchDragStack Laxx;
+	FIELD B touchDropPositionX I
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/ContainerScreen.mapping]: Why not just touchDropX/touchDropY? There is such a thing as "too verbose".

File: mappings/net/minecraft/client/gui/RotatingCubeMapRenderer.mapping [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CLASS dgo net/minecraft/client/gui/RotatingCubeMapRenderer
+	FIELD a minecraftClient Lcqx;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/RotatingCubeMapRenderer.mapping]: client

**Review by asiekierka — CHANGES_REQUESTED** [review]
Too verbose, in general!

**Comment by holly-hacker** [comment]
I'd rather have the user instantly understand what they're reading than have them guess but read a few characters less, though I agree with you on the ones you mentioned. Fixed those.

**Review by Gegy — APPROVED** [review]

**Review by asiekierka — APPROVED** [review]

**Pull request merged**